frontend/src/features/notes/list/NoteResults/NoteResults.tsx

```diff
@@ -0,0 +1,84 @@
+import { IconButton } from 'components/common/buttons';
+import { ColumnWithProps, DateCell, Table } from 'components/Table';
+import { TableSort } from 'components/Table/TableSort';
+import { Api_Note } from 'models/api/Note';
+import { MdDelete } from 'react-icons/md';
+import { CellProps } from 'react-table';
+
+export interface INoteResultProps {
+  results: Api_Note[];
+  loading?: boolean;
+
+  onDelete: (note: Api_Note) => void;
+  sort: TableSort<Api_Note>;
+  setSort: (value: TableSort<Api_Note>) => void;
+}
+
+export function NoteResults(props: INoteResultProps) {
+  const { results, setSort, sort, ...rest } = props;
+
+  const columns: ColumnWithProps<Api_Note>[] = [
+    {
+      Header: 'Note',
+      accessor: 'note',
+      align: 'left',
+      sortable: false,
+      minWidth: 100,
+      width: 100,
+    },
+    {
+      Header: 'Created date',
+      accessor: 'appCreateTimestamp',
+      align: 'center',
+      sortable: true,
+      width: 10,
+      maxWidth: 20,
+      Cell: DateCell,
+    },
+    {
+      Header: 'Last updated by',
+      accessor: 'appLastUpdateUserid',
+      align: 'center',
+      sortable: true,
+      width: 10,
+      maxWidth: 20,
+    },
+
+    {
+      Header: '',
+      accessor: 'id',
+      align: 'right',
+      sortable: false,
+      width: 10,
+      maxWidth: 20,
+      Cell: (cellProps: CellProps<Api_Note>) => {
+        return (
+          <IconButton
+            title="Delete Note"
+            variant="light"
+            onClick={() => {
+              props.onDelete(cellProps.row.original);
+            }}
+          >
+            <MdDelete size={22} />
+          </IconButton>
+        );
+      },
+    },
+  ];
+  return (
+    <Table<Api_Note>
+      name="notesTable"
+      manualSortBy={false}
+      pageSize={-1}
```

Comment:
not sure we need to pass "-1" here? I think there was a hidePagination prop you could use?

Reply:
We need to enhance the Table component to give flexibility to hide the paging tool bar, currently when we hide it it does not work correctly